### DISCUSSION TITLE
Development will be (mostly) halted coming months

### DISCUSSION THREAD

**Discussion by AIrjen** [AIrjen/OneButtonPrompt#123] [Announcements]
Hi everyone,

I wanted to let you know that development will me mostly halted coming months. We've got some positive personal stuff to attend to, and in this period I will not have access to my main computer for a while. Which means I can't run SD and try stuff out.

This means I can't add/change or test things for a while.

I am very happy with the current state of One Button Prompt, and it is already very mature. It's even now supported in both A1111 and ComfyUI. At the moment of writing, its also issue-free. Let's hope noone finds a breaking bug soon. ;)

I will still be able to approve any incoming Pull Request (but not test). I will still be active answering question and gathering ideas. So I will still be monitoring the GitHub, reddit and discord. Just from my phone.

There are still many things on the todo-list I'd like to add. And maybe sometimes I can sneak an hour in ;)

AIrjen

**Comment by SirMathelot** [comment]
Thank you for the great extension! Have a nice break :)
